src/main/java/uk/ac/ebi/embl/converter/fftogff3/FFGFF3HeadersFactory.java

```diff
@@ -0,0 +1,29 @@
+package uk.ac.ebi.embl.converter.fftogff3;
+
+import uk.ac.ebi.embl.api.entry.Entry;
+import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.converter.IConversionRule;
+import uk.ac.ebi.embl.converter.gff3.GFF3Headers;
+
+public class FFGFF3HeadersFactory implements IConversionRule<Entry, GFF3Headers> {
+
+    @Override
+    public GFF3Headers from(Entry entry) {
+
+        for (Feature feature : entry.getFeatures().stream().sorted().toList()) {
```

Comment:
Suggestion to  get the source Feature using 
`Feature sourceFeature = entry.getPrimarySourceFeature();`